### PR TITLE
Create 2020-10-19-solidity-0.7.4-release-announcement.md

### PR DESCRIPTION
**Pull request by franzihei**

### CODE REVIEW

**Comment by chriseth** [comment]
Added some more text.

File: _posts/2020-10-19-solidity-0.7.4-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+layout: post
+published: true
+title: 'Solidity 0.7.4 Release Announcement'
+date: '2020-10-19'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.7.4](https://github.com/ethereum/solidity/releases/tag/v0.7.4) fixes a storage corruption bug of medium severity.
+To learn more about the bug and check if your contract is vulnerable please read this [post](https://solidity.ethereum.org/2020/10/19/empty-byte-array-copy-bug) with
```
Comment (hrkrshnn) [inline _posts/2020-10-19-solidity-0.7.4-release-announcement.md:11]:
```suggestion
To learn more about the bug and to check if your contract is vulnerable please read this [post](https://solidity.ethereum.org/2020/10/19/empty-byte-array-copy-bug) with
```

File: _posts/2020-10-19-solidity-0.7.4-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+struct Fixed { uint value; }
+
+function toFixed(uint x) pure returns (Fixed memory ret) {
+    ret.value = x * MULTIPLIER;
+}
+```
+
+Such constants have to follow the same rules as constants at contract-level.
+This means that for example the value to be fixed at compile-time and not
+all types are supported.
```
Comment (hrkrshnn) [inline _posts/2020-10-19-solidity-0.7.4-release-announcement.md:40]:
```suggestion
This means, for example, the value has to be fixed at compile-time and only
some types are supported.
```

File: _posts/2020-10-19-solidity-0.7.4-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+You can import constants from other source files as you import any other identifier:
```
Comment (hrkrshnn) [inline _posts/2020-10-19-solidity-0.7.4-release-announcement.md:42]:
```suggestion
You can import constants from other source files like you import any other identifier:
```

File: _posts/2020-10-19-solidity-0.7.4-release-announcement.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+You can import constants from other source files like you import any other identifier:
+
+```import {MyConstant, MyStruct} from "myHelper.sol";```
+
+
+
+## Full Changelog
+
+### Language Features:
```
Comment (chriseth) [inline _posts/2020-10-19-solidity-0.7.4-release-announcement.md:51]: There were some more changes - please take this version: https://github.com/ethereum/solidity/pull/10065

File: _posts/2020-10-19-solidity-0.7.4-release-announcement.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+}
+```
+
+Such constants have to follow the same rules as constants at contract-level.
+This means, for example, the value has to be fixed at compile-time and only
+some types are supported.
+
+You can import constants from other source files like you import any other identifier:
+
+```import {MyConstant, MyStruct} from "myHelper.sol";```
```
Comment (chriseth) [inline _posts/2020-10-19-solidity-0.7.4-release-announcement.md:42]:
```suggestion
``import {MyConstant, MyStruct} from "myHelper.sol";``
```

**Review by hrkrshnn — APPROVED** [review]

**Review by franzihei — COMMENTED** [review]
ok to go from my side

**Pull request merged**